### DISCUSSION TITLE
[reCAPTCHA] is there is possible way to disable reCAPTCHA login with phone number?

### DISCUSSION THREAD

**Discussion by kristoff2016** [invertase/react-native-firebase#4613] [Q&A] [answered]
Thank you so much for the great library,

I was wondering if there is any way that I could hide reCAPTCHA step? 

Thank you so much!

**Comment by andersonaddo** [comment, marked as answer]
Nope, there isn't.
https://github.com/invertase/react-native-firebase/discussions/4183
